puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -119,7 +119,8 @@
       # $_volume_clone_opts = "interleave=true"
       # $_cinder_volume_resource_name = "cinder-volume-clone"
     } else {
-      $_volume_clone_opts = undef
+      #$_volume_clone_opts = undef
```

Comment:
Agreed.  I left these in originally in case there was expectation of eventually returning to the "better" syntax (that doesn't work on some releases) somewhere down the line, but cleaning these out is the better choice and just worry about it later.